kartograf/merge.py

```diff
@@ -1,20 +1,52 @@
 import ipaddress
 import shutil
-import numpy as np
-from pandarallel import pandarallel
 import pandas as pd
 
 from kartograf.timed import timed
 
+class Networks:
+    '''
+    A class whose _dict represents a mapping of the network number and IP networks within that network for a given AS file.
+
+    To check inclusion of a given IP network in the base AS file, we can compare (see check_inclusion) the networks under the root network number instead of all the networks in the base file.
+    '''
+    def __init__(self):
+        self._dict = dict()
+        self._keys = self._dict.keys()
+        for i in range(0, 255):
+            self._dict[i] = []
+
+    def update(self, pfx):
+        ipn = ipaddress.ip_network(pfx)
+        netw = int(ipn.network_address)
+        mask = int(ipn.netmask)
+        if ipn.version == 4:
+            root_net = int(str(pfx).split(".")[0])
```

Comment:
yea exactly, it would make it hard to figure out any range over /8. The next optimization for this, if needed, would be a trie indexed by octet i think.